cmd/object-lock.go

```diff
@@ -58,6 +58,27 @@ func parseRetentionMode(modeStr string) (mode RetentionMode) {
 	return mode
 }
 
+// LegalHoldStatus - object legal hold status.
+type LegalHoldStatus string
+
+const (
+	// ON -legal hold is on.
+	ON LegalHoldStatus = "ON"
+
+	// OFF -legal hold is off.
+	OFF LegalHoldStatus = "OFF"
+)
```

Comment:
I am getting an impression that we should move the code here into either pkg/objectlock or cmd/objectlock helps cleanup quite a few things and add unit tests..

Reply:
this is done